packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -10,20 +10,86 @@ import 'package:flutter/widgets.dart';
 
 import 'colors.dart';
 import 'constants.dart';
+import 'text_theme.dart';
 import 'theme.dart';
 
-// Measured against iOS 12 in Xcode.
-const EdgeInsets _kButtonPadding = EdgeInsets.all(16.0);
-const EdgeInsets _kBackgroundButtonPadding = EdgeInsets.symmetric(
-  vertical: 14.0,
-  horizontal: 64.0,
-);
 
-// The relative values needed to transform a color to it's equivalent focus
-// outline color.
-const double _kCupertinoFocusColorOpacity = 0.80;
-const double _kCupertinoFocusColorBrightness = 0.69;
-const double _kCupertinoFocusColorSaturation = 0.835;
+// Measured against iOS (17) [Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/buttons#iOS-iPadOS).
+
+/// The size of a [CupertinoButton].
+/// Based on the iOS (17) [Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/buttons#iOS-iPadOS).
```

Comment:
```suggestion
/// The size of a [CupertinoButton].
///
/// Based on the iOS (17) [Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/buttons#iOS-iPadOS).
```
Dart doc comments need a line between the first sentence and the rest. The first line is a short summary that shows up as a preview when the detailed view isn't needed. It makes it easier to read in the code when there's a gap. This needs to happen throughout this PR.